.github/workflows/all_tests_cron.yml

```diff
@@ -16,25 +20,25 @@ jobs:
       - name: Get dependencies
         run: |
           bash scripts/gh-action-get-deps.sh
-  unit_tests:
-    name: Run all unit tests
-    runs-on: ubuntu-latest
-    needs: dependencies
-    strategy:
-      matrix:
-        app: [andi, discovery_api]
-    steps:
       - name: Run unit tests
         run: |
           bash scripts/gh-action-unit-test.sh ${{ matrix.app }}
   integration_tests:
     name: Run all integration tests
     runs-on: ubuntu-latest
-    needs: dependencies
     strategy:
       matrix:
         app: [andi, discovery_api, pipeline, e2e]
     steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-elixir@v1.5.0
+        with:
+          otp-version: 21.3
+          elixir-version: 1.10.4
+          experimental-otp: true
+      - name: Get dependencies
+        run: |
+          bash scripts/gh-action-get-deps.sh
```

Comment:
The integration test are failing locally for me, but they also are failing for other files locally. Hoping this works?